graphrag/query/structured_search/local_search/mixed_context.py

```diff
@@ -484,6 +492,8 @@ def _build_local_context(
             final_context_data = current_context_data
 
         # attach entity context to final context
+        if graphdb_client:
+            graphdb_client._client.close()
```

Comment:
I don't know if you want to close it here either b/c we might extend the mixed_context to stay alive to have multiple questions. At least the # attach entity context... comment should be above what it is commenting about